### PR TITLE
Fix LICENSE.md link

### PR DESCRIPTION
**Pull request by azu**
Follow up https://github.com/ContributorCovenant/contributor_covenant/pull/371 https://github.com/ContributorCovenant/contributor_covenant/pull/372

### CODE REVIEW

**Comment by CoralineAda** [comment]
We're actually under a creative commons license now. Do you want to update that while you're at it? Thanks!

File: adopters/index.html [hunk 1]
```diff
@@ -203,7 +203,7 @@
 	</section>
 
 	<footer id="footer">
-		The Contributor Covenant was created by <a href="http://where.coraline.codes/" title="Coraline Ada Ehmke">Coraline Ada Ehmke</a> in 2014 and is released under an <a href="https://github.com/ContributorCovenant/contributor_covenant/blob/master/LICENSE">MIT license</a>.<br>
+		The Contributor Covenant was created by <a href="http://where.coraline.codes/" title="Coraline Ada Ehmke">Coraline Ada Ehmke</a> in 2014 and is released under an <a href="https://github.com/ContributorCovenant/contributor_covenant/blob/master/LICENSE.md">MIT license</a>.<br>
```
Comment (azu) [inline adopters/index.html]: Ok. I will change `MIT license` to `Creative Commons Attribution 4.0 International Public License.`.

**Comment by CoralineAda** [comment]
♥️

**Comment by azu** [comment]
done

**Pull request merged**